README.md

```diff
@@ -15,159 +15,5 @@ tightly tailored to the needs of our team members working on various projects wi
 - https://devenv.sh/
 - https://developer.shopware.com/docs/guides/installation/devenv
 
-## Setup
```

Comment:
Can we add a "look at the wiki" hint here 🙂 .

Reply:
Good point. What's your opinion about the other menu entries moved from the readme to docs?

Reply:
Sorry, I meant all points - for the sake of simplicity I had only picked out the top one 🙂 

I think that through your change, the wiki is now present enough 👍

Reply:
I moved "Setup & Usage" one up to make in even more present